cmd/ocsp-updater/main.go

```diff
@@ -422,6 +440,13 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(ctx context.Context, batchSize
 		return err
 	}
 
+	for _, s := range statuses {
```

Comment:
Oh blah. I don't know how I confused myself here - you're both right. I think I confused myself on the OCSP response being expired vs the certificate somehow and thought I could process the results from this existing query.

Now I think I might need to make an entirely separate `looper` with its own `dbMap.Select` to find expired certificates. Right now the utility finds `certificateStatus` rows that are for:
a) revoked certificates that need a revoked OCSP status generated
b) certificates without a OCSP status
c) certificates that haven't expired, but have a stale OCSP response

None of the above will be sufficient for finding expired certificates.

Thoughts? Generally the right direction?